CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 * Update rubocop to `0.73.0`. ([@r.dubrovsky][])
 * Use prefered variable name in rescued exceptions (cop: `Naming/RescuedExceptionsVariableName`). ([@ula][])
 * Disable `RSpec/ImplicitSubject` cop for rspec files. ([@r.dubrovsky][])
+* Write empty methods in two lines ([@theblackarrovv][])
```

Comment:
add pls note to the (unreleased block). New changes are not a part of the last version.